DQMOffline/LumiZCounting/interface/RunLumiRangeMap.h

```diff
@@ -0,0 +1,26 @@
+#ifndef DQMOFFLINE_LUMIZCOUNTING_RUNLUMIRANGEMAP_H
+#define DQMOFFLINE_LUMIZCOUNTING_RUNLUMIRANGEMAP_H
+
+#include <string>
+#include <vector>
+#include <map>
+
+namespace baconhep
+{
+  class RunLumiRangeMap
+  {
+    public:
+      typedef std::pair<unsigned int, unsigned int>                RunLumiPairType;
+      typedef std::map<unsigned int, std::vector<RunLumiPairType>> MapType;
+      
+      RunLumiRangeMap(){}
```

Comment:
Not for now, will remove to test/ since it might be used if we decide to do a lumi mask in the future.